packages/atomic/src/components/result-template-components/atomic-result-link/atomic-result-link.tsx

```diff
@@ -15,10 +16,37 @@ import {filterProtocol} from '../../../utils/xss-utils';
 export class AtomicResultValue {
   @ResultContext() private result!: Result;
 
+  @Element() private host!: HTMLElement;
+
+  /**
+   * Where to display the linked URL, as the name for a browsing context (a tab, window, or <iframe>).
+   *
+   * The following keywords have special meanings for where to load the URL:
+   * - _self: the current browsing context. (Default)
+   * - _blank: usually a new tab, but users can configure browsers to open a new window instead.
+   * - _parent: the parent browsing context of the current one. If no parent, behaves as _self.
+   * - _top: the topmost browsing context (the "highest" context that’s an ancestor of the current one). If no ancestors, behaves as _self.
+   */
+  @Prop() target = '_self';
+
+  private hasSlot!: boolean;
+
+  public connectedCallback() {
+    this.hasSlot = !!this.host.children.length;
+  }
+
   public render() {
     return (
-      <a part="result-link" href={filterProtocol(this.result.clickUri)}>
-        <atomic-result-text field="title"></atomic-result-text>
+      <a
+        part="result-link"
+        href={filterProtocol(this.result.clickUri)}
+        target={this.target}
+      >
+        {this.hasSlot ? (
```

Comment:
Would it be simpler to check `this.host.children.length` here and remove the `hasSlot` property and `connectedCallback` method?

Reply:
If there's a subsequent re-render, hasSlot will always be true